clients/memaslap.c

```diff
@@ -289,6 +299,26 @@ static const char *ms_lookup_help(ms_options_t option)
       "UDP support, default memslap uses TCP, TCP port and UDP port of\n"
       "        server must be same.";
 
+  case OPT_TLS:
+    return
+      "Use TLS, certificate and private key are must";
+
+  case OPT_TLS_CERT:
+    return
+      "Path to TLS certificate file. must be with '--tls' option";
+
+  case OPT_TLS_KEY:
+    return
+      "Path to TLS private key file. must be with '--tls' option";
+
+  case OPT_TLS_CA:
+    return
+      "Path to CA file. optional. must be together with '--tls' option";
+
+  case OPT_TLS_SKIP_VERIFY:
+    return
+      "Skip TLS peer verification. must be together with '--tls' option";
```

Comment:
peer hostname verification

Reply:
There is two possible 'skips' - one (skip_cert_verify) skips the server's certificate verification (good for testing), and the second skip (skip_hostname_verify) is for the hostname. you can see these configs in libmemcached-1.0/struct/tls.h.
This file - clients/memaslap.c isn't apart of the libmemcached code, it's a client that uses libmemcached which I added to the tls functionality. I will add the missing skip_hostname to this client too, but just to be clear that this client has no impact on the php-memcached client.

Reply:
Can you rename this to be `OPT_TLS_SKIP_HOSTNAME_VERIFY`?

Reply:
changed it to OPT_TLS_SKIP_CERT_VERIFY and added OPT_TLS_SKIP_HOSTNAME_VERIFY